pyscada/visa/devices/Keithley_DMM2000.py

```diff
@@ -1,36 +1,101 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
 from pyscada.visa.devices import GenericDevice
+from pyscada.models import DeviceWriteTask, Variable, Device
+from pyscada.models import RecordedData
+import time
+import math
+
+import logging
 
+logger = logging.getLogger(__name__)
 
 class Handler(GenericDevice):
-    """
+    '''
```

Comment:
please leave the `"""` doc strings see [PEP 257](https://www.python.org/dev/peps/pep-0257/)